website/source/docs/guides/leader-election.html.md

```diff
@@ -1,130 +1,141 @@
+`?acquire=<session>` query parameter. 
+
+The `<body>` of the PUT should be a
+JSON object representing the local instance. This value is opaque to
 Consul, but it should contain whatever information clients require to
 communicate with your application (e.g., it could be a JSON object
 that contains the node's name and the application's port).
 
-Attempt to `acquire` the `<key>`. This will look something like (note that
-`<session>` is the ID returned by the call to
-[`/v1/session/create`](/api/session.html#session_create)):
-
-```text
-curl -X PUT -d <body> http://localhost:8500/v1/kv/<key>?acquire=<session>
+```sh
+$ curl -X PUT -d <body> http://localhost:8500/v1/kv/lead?acquire=ca8e74b-6350-7587-addf-a18084928f3c
  ```
 
 This will either return `true` or `false`. If `true`, the lock has been acquired and
-the local node is now the leader. If `false` is returned, some other node has acquired
+the local service instance is now the leader. If `false` is returned, some other node has acquired
 the lock.
 
-All nodes now remain in an idle waiting state. In this state, we watch for changes
-on `<key>`. This is because the lock may be released, the node may fail, etc.
+### Watch the Session 
+
+All instances now remain in an idle waiting state. In this state, they watch for changes
+on key `dbservice`. This is because the lock may be released or the instance could fail, etc.
+
 The leader must also watch for changes since its lock may be released by an operator
 or automatically released due to a false positive in the failure detector.
 
 By default, the session makes use of only the gossip failure detector. That
 is, the session is considered held by a node as long as the default Serf health check
 has not declared the node unhealthy. Additional checks can be specified if desired.
 
-Watching for changes is done via a blocking query against `<key>`. If we ever
-notice that the `Session` of the `<key>` is blank, there is no leader, and we should
+Watching for changes is done via a blocking query against `<key>`. If they ever
+notice that the `Session` of the `<key>` is blank, there is no leader, and then should
 retry lock acquisition. Each attempt to acquire the key should be separated by a timed
 wait. This is because Consul may be enforcing a [`lock-delay`](/docs/internals/sessions.html).
 
+### Release the Session
+
 If the leader ever wishes to step down voluntarily, this should be done by simply
 releasing the lock:
 
-```text
-curl -X PUT http://localhost:8500/v1/kv/<key>?release=<session>
+```sh
+$ curl -X PUT http://localhost:8500/v1/kv/dbservice?release=4ca8e74b-6350-7587-addf-a18084928f3c
 ```
 
-## Discovering a Leader
+## Discover the Leader
 
-Another common practice regarding leader election is for nodes to wish to identify the
-leader for a given service.
+Another common practice regarding leader election is for non-leader instances may wish to identify the leader for the given set of service instances.
```

Comment:
This is a bit odd to read. Maybe it should be something like:
> Another common practice regarding leader election is that non-leader instances may wish to identify the leader for the given set of service instances.

Reply:
I've completely re-worked this opening.